themes/Atomize-color-theme.json

```diff
@@ -884,6 +884,26 @@
         "foreground": "#C678DD"
       }
     },
+    {
+      "name": "C++ Namespaces",
+      "scope": [
+        "entity.name.scope-resolution.parameter.cpp",
+        "entity.name.scope-resolution.cpp",
+        "entity.name.scope-resolution.function.call.cpp",
+        "entity.name.scope-resolution.template.call.cpp",
+        "entity.name.namespace"
+      ],
+      "settings": {
+        "foreground": "#E5C07B"
+      }
+    },
+    {
+      "name": "C++ Properties",
+      "scope": ["variable.other.property"],
+      "settings": {
+        "foreground": "#BE5046"
+      }
+    },
```

Comment:
What did this fix for you? Since it's not suffixed with `.cpp`, it affects all languages and is breaking javascript syntax:

<img width="257" alt="Screen Shot 2020-08-24 at 10 21 11" src="https://user-images.githubusercontent.com/12487270/91056136-93efc780-e5f3-11ea-9917-5f26402b0472.png">

Reply:
Hm... In C++ `variable.other.property` is class members. I just set a slightly different color for it as in Qt Creator.
Is there a way to define such highlighting only for C++?

Reply:
This does not work for class members for me. Are you using an extension that changes the syntax highlighting of c++? You might be able to simply do `variable.other.property.cpp`. If you use the "Inspect Editor Tokens and Scope" tool in vscode and click on a class member, it'll tell you its scope.

Either way though, I would probably use the same color as variables (`#E06C75`) for this to be consistent with other languages.

Reply:
> This does not work for class members for me. Are you using an extension that changes the syntax highlighting of c++? 

I use [official C/C++ extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools) from Microsoft. 

> You might be able to simply do variable.other.property.cpp. If you use the "Inspect Editor Tokens and Scope" tool in vscode and click on a class member, it'll tell you its scope.

Yes, that's exactly what I use, but it does not show `cpp` suffux :(
![изображение](https://user-images.githubusercontent.com/22453358/91070812-fbcbff80-e63f-11ea-80ce-f6fe6ee81230.png)

Reply:
In C++ highlighting class members in a different colors very useful. But if this is not possible to do without impact on other languages I will undo this change.

Reply:
Oh strange. But yeah unfortunately, I'd rather not use a new color for stuff in C++ and keep things consistent between languages. In JavaScript/TypeScript, class members are also the same color as variables. Are they the right color (the same as variables) in C++ right now without this change?

Reply:
Yes. I just tried to make color more distinguishable as in Qt Creator and other C++ IDE. This allows you to see if a variable is a member of a class or a regular variable.

Reply:
Great, so do you mind removing this section and keeping class members the same color as variables?

Reply:
Sure.